.github/workflows/ci.yml

```diff
@@ -40,8 +37,7 @@ jobs:
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v4
         with:
-          python-version: "3.10"
-        # See https://github.com/nodejs/node-gyp/issues/1574#issuecomment-554187531
```

Comment:
This comment is still relevant and applies to the "Configure macOS env" step below.

Reply:
I've moved this to the `binding.gyp` fixture so tests can be run locally on macOS, thanks for highlighting it.